sematic/db/models/base.py

```diff
@@ -1,8 +1,8 @@
 # Third-party
-from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import DeclarativeBase
```

Comment:
`declarative_base` decorator no longer exists in 2.0; `DeclarativeBase` is required instead